src/csync/vio/csync_vio_local_win.cpp

```diff
@@ -189,12 +196,29 @@ std::unique_ptr<csync_file_stat_t> csync_vio_local_readdir(csync_vio_handle_t *d
 
     file_stat->size = (handle->ffd.nFileSizeHigh * ((int64_t)(MAXDWORD)+1)) + handle->ffd.nFileSizeLow;
     file_stat->modtime = FileTimeToUnixTime(&handle->ffd.ftLastWriteTime, &rem);
+
+    std::wstring fullPath;
+    fullPath.reserve(std::wcslen(handle->path) + std::wcslen(handle->ffd.cFileName));
+    fullPath += handle->path;
+    fullPath += handle->ffd.cFileName;
```

Comment:
A backslash will never be needed here?
Also wstring should be UTF-16, so I think you could also use `QString`.

Reply:
I could use QString but then i need to reinterpret_cast to wchar_t.

I don't understand your comment about backslash.  handle->path; always end with a slash, by contruction. I'll add a comment